<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SvgResizingView" name="gameView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStartMap"/>
    <addaction name="actionStartCampaign"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionCreateMap"/>
    <addaction name="actionEditMap"/>
    <addaction name="actionStartTestGame"/>
    <addaction name="actionStopTestGame"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuEditor"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionStartMap">
   <property name="text">
    <string>Start Map</string>
   </property>
  </action>
  <action name="actionStartCampaign">
   <property name="text">
    <string>Start Campaign</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStartMapEditor">
   <property name="text">
    <string>Start Map Editor</string>
   </property>
  </action>
  <action name="actionCreateMap">
   <property name="text">
    <string>Create Map</string>
   </property>
  </action>
  <action name="actionEditMap">
   <property name="text">
    <string>Edit Map</string>
   </property>
  </action>
  <action name="actionStartTestGame">
   <property name="text">
    <string>Start Test Game</string>
   </property>
  </action>
  <action name="actionStopTestGame">
   <property name="text">
    <string>Stop Test Game</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SvgResizingView</class>
   <extends>QGraphicsView</extends>
   <header>core/svgresizingview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
